text_generation/causal_lm/cpp/continuous_batching/library/src/sequence_group.hpp

```diff
@@ -445,6 +465,9 @@ class SequenceGroup {
                 for (auto& sequence : m_sequences) {
                     // todo: check seq.is_finished() to generate without several </s>
                     // or is it ok to use padding?
+                    if (!sequence->get_generated_len()) {
```

Comment:
Is this check added for OOM situation? Because, correct me if I'm wrong, we can enter this block only if we can't finish prompt processing and notify is called from out of memory segment right?

Reply:
It was w/a for case of assert in sequence->get_last_generation_output(); @mzegla Please provide the fix

Reply:
Okay, for now I think it's good. We could check for out_of_memory or abort earlier in notify and don't enter this block at all. I'll propose something like that in another PR.